### DISCUSSION TITLE
IntelliJ IDEA Based container instead of VSCode?

### DISCUSSION THREAD

**Discussion by nooobcoder** [coder/code-server#2673] [Q&A] [answered]
Would there be an IntelliJ IDEA based code-server instead of VSCode. I personally like to use IDEA for Java development. Is there any testing channel to have a  try of containerized IDEA by **coder** ?

Any suggestions, are welcome and thanks in advance.

**Comment by nooobcoder** [comment]
@code-asher @nhooyr any reply on this?
I did not mean an enterprise version, I am already aware of the enterprise version but I need a code-server that is based on IntelliJ IDEA.

**Comment by code-asher** [comment, marked as answer]
Currently I don't believe we have any timeline for releasing this
outside enterprise but I think it might happen eventually.
